pep.rst

```diff
@@ -484,6 +488,23 @@ the following, at the cost of losing some type specificity:
     def mytag(*args: str | tuple) -> Any:
         ...
 
+A user might write a tag string as a split string:
```

Comment:
Let's revisit this part of the specification. The current problem here I mention is that
```
tag"\N{{GRINNING FACE}}"
```
is called with
```
tag('\\N{', 'GRINNING FACE}')
```
so one raw string is resulting into two str args. We want the specification to disallow this split. and therefore it's an implementation bug, not just one possible way to implement the spec.

So something like the following should work:
> Any string parts of the tag string can only be split by an interpolation.

And then we can show this example.